app/src/release/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugModule.kt

```diff
@@ -6,5 +6,6 @@ import javax.inject.Inject
 class DebugModule @Inject constructor() {
     fun sendCryingProbabilityEvent(cryingProbability: Float) {}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/8b49d288c8cda524f66973edfcfe165e7a30812e/app/src/release/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugModule.kt#L7"></span>Function parameter cryingProbability is unused.</td>
    </tr>
  </tbody>
</table>